packages/runtime/test/src/runtime.ts

```diff
@@ -1441,13 +1442,56 @@ describe("Helper functions", function () {
 		});
 	});
 });
-describe("Funtions related to runtime chain", function () {
+
+describe("Atomic Transaction Composer",function () {
 	let runtime: Runtime;
+	const notABI = "not-ABI.json";
+	const notJSON = "not-JSON.teal";
+	const networkRuntimeNotDefined = "network-runtime-undefined.json";
+	const correctABI = "correct-ABI.json";
+	useFixture("atomic-transaction-composer")
 
 	this.beforeEach(function () {
 		runtime = new Runtime([]);
 	});
 
+	it("Should throw an error if file is not found", function () {
+		assert.throws(() => {
+            runtime.parseABIContractFile("doesNotExist.json");
+		});
+	});
+  
+	it("Should throw an error if file is not ABI", function () {
+		assert.throws(() => {
+			runtime.parseABIContractFile(notABI);
+		})
+	});
+  
+	it("Should throw an error if file is not json", function () {
+		assert.throws(() => {
+			runtime.parseABIContractFile(notJSON);
+		});
+	});
+  
+	it("Shoult from an error if network.runtime is not defined", function () {
```

Comment:
Would be great, if we can create runtime error for all these scenarios. It would be easy to catch them.